src/main.ts

```diff
@@ -165,6 +169,45 @@ const command: Command = {
 		else {
 			return compile(config(configArgs), options);
 		}
+	},
+	eject(helper: Helper) {
+		const ejectOutput: EjectOutput = {
+			npm: {
+				devDependencies: {
```

Comment:
Ok, fair. So we'd need a way to support pulling from package.json OR specifying a specific semver in the case of `cli-` dependencies that aren't in the package.json. @agubler Would you see this as something that the `cli-eject` command should handle, ie. pass the package.json along as part of `eject` output or should each command handle it itself?